packages/commonjs/src/helpers.js

```diff
@@ -1,6 +1,17 @@
 export const PROXY_SUFFIX = '?commonjs-proxy';
 export const getProxyId = (id) => `\0${id}${PROXY_SUFFIX}`;
-export const getIdFromProxyId = (proxyId) => proxyId.slice(1, -PROXY_SUFFIX.length);
+
+export const PROXY_NAMED_SUFFIX = '?commonjs-nproxy';
```

Comment:
This looked like a typo, then I realized it's not. Maybe a more distinct name would be better.